src/mcsat/value.h

```diff
@@ -41,15 +43,15 @@ typedef enum {
   VALUE_BV
 } mcsat_value_type_t;
 
-typedef struct value_s {
+struct mcsat_value_s {
```

Comment:
is this typedef done somewhere else?

Reply:
did move the typedef to `mcsat_types.h` (iirc to be aligned with all other definitions in that file)